server/routes/departments.js

```diff
@@ -20,6 +20,10 @@ router.get('/departments/:departmentId', co.wrap(function* (req, res) {
 }));
 
 router.post('/departments', co.wrap(function* (req, res) {
+    if (!permissionsUtils.isAdmin(req.userDetails)){
```

Comment:
this should be accessible to admins only

Reply:
@may-ben-arie - it is, that's why you have this IF - if it's not the admin, it returns an error with 403 http code